examples/optimistic_locking.py

```diff
@@ -46,7 +47,8 @@ def assert_condition_check_fails():
     except TransactWriteError as e:
         assert isinstance(e.cause, ClientError)
         assert e.cause_response_code == "TransactionCanceledException"
-        assert "ConditionalCheckFailed" in e.cause_response_message
+        if e.cause_response_message is not None:
+            assert "ConditionalCheckFailed" in e.cause_response_message
```

Comment:
```suggestion
        assert e.cause_response_message is not None
        assert "ConditionalCheckFailed" in e.cause_response_message
```
Otherwise you're silently succeeding if unexpected `e.cause_response_message is None` happens.